cmd/airflow.go

```diff
@@ -31,6 +31,7 @@ var (
 	envFile                string
 	customImageName        string
 	settingsFile           string
+	composeFile            string
```

Comment:
you would need to maintain two separate variables for compose file because you are using it in two separate commands, and one of them has default value and the other one doesn't have, but cobra library isn't smart enough to figure out where to use the default and where not to if it is on the same variable
```suggestion
	composeFile            string
	exportComposeFile string
```